File: Gym-Website/src/app/Model/OrderItem.ts
export class OrderItem{
    image:string;
    name:string;
    price:string;
    quantity:string;

    constructor( image:string, name:string, price:string, quantity:string){
        this.image=image;
        this.name=name;
        this.price=price;
        this.quantity=quantity
    }
}

File: Spring/src/main/java/com/example/springboot/Model/OrderItem.java
package com.example.springboot.Model;

import javax.persistence.*;

@Entity
public class OrderItem {
    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
    private long id;

    @ManyToOne
    private Product product;

    @ManyToOne
    @JoinColumn(name="order_id")
    private Order order;

    private int quantity;

    public OrderItem() {
    }

    public OrderItem(Product product, int quantity) {
        this.product = product;
        this.quantity = quantity;
    }

    public Product getProduct() {
        return product;
    }

    public int getQuantity() {
        return quantity;
    }
}
